src/messages/MOSDMarkMeDown.h

```diff
@@ -54,6 +61,8 @@ class MOSDMarkMeDown final : public PaxosServiceMessage {
     decode(target_addrs, p);
     decode(epoch, p);
     decode(request_ack, p);
+    assert(header.version >= 4);
```

Comment:
this should be a conditional - consider what happens if an older osd sends the older version of this message to a monitor.

Rather than asserting, we only decode the new value when the message is the appropriate version, otherwise the default of false will apply, which will work fine.

Reply:
Done